iup/__init__.py

```diff
@@ -76,6 +75,22 @@ def with_columns(self, *args, **kwargs):
         result = super().with_columns(*args, **kwargs)
         return orig_class(result)
 
+    @staticmethod
```

Comment:
It would be nice to have all these `assert_something` methods as consistent as possible - at least all static methods or all normal methods. Could `assert_column_name_all` take `self` as its first argument and remove the "date" column internally?

Reply:
Good point! Changed in 6f6f54b21f3c366380898cf70c259e3e0e389c76

Reply:
Great point! See change in 8ead6e304c8f18304e32fbaa24b7f5b216ea247c